### PR TITLE
Document Sentry issue actions

### PR DESCRIPTION
**Pull request by ChrisBAshton**
We ought to document how to merge, resolve and ignore issues, and when it is appropriate to do so.

### CODE REVIEW

File: source/manual/sentry.html.md [hunk 1]
```diff
@@ -224,3 +224,46 @@ are configured in the [Alerts panel][].
...
+less noisy, but also has the advantage of marking it as a regression and emailing
+you if the issue occurs again.
+
+### Ignoring an issue
+
+When you've identified an issue and written it up as a Trello card, or are
+actively working on fixing it, it can be unhelpful for the issue to keep
+accumulating events (and potentially spamming your Slack channel). In these cases,
+you should [comment on the issue][anchor-comment] with a link to your card or PR,
+then ignore the issue.
```
Comment (thomasleese) [inline source/manual/sentry.html.md:259]: Do we want to think about using the "Assignee" feature in Sentry as another way to determine if the issue is being worked on?

Comment (ChrisBAshton) [reply source/manual/sentry.html.md:259]: We could do. I haven't used it a lot yet, as we already have visibility over that sort of thing on Trello, but it seems sensible to ask people to assign themselves to issues they're working on too. I'll add a line.

Comment (thomasleese) [reply source/manual/sentry.html.md:259]: Cool, sounds good. It does feel like in many ways we're underutilising the features of Sentry, such as the performance measurements, automatic creation of GitHub/Trello cards, assigning apps to various teams, etc, etc... but this is definitely an improvement over what we had before!

Comment (ChrisBAshton) [reply source/manual/sentry.html.md]: Yep, I'm hoping to set up a new GitHub integration (we have one already, but it's legacy) and start properly linking Sentry issues either with the git commit of the corresponding code, or the release tag. And hopefully having it notify the code author if there is an issue.

This PR should be ready for re-review btw :)

**Review by thomasleese — APPROVED** [review]

**Pull request merged**